handsontable/src/core.js

```diff
@@ -4713,11 +4713,8 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
   };
 
   const shortcutManager = createShortcutManager({
-    handleEvent(event) {
-      const isListening = instance.isListening();
-      const isKeyboardEventWithKey = event?.key !== void 0;
```

Comment:
@wszymanski I removed this line as I felt it's here only for tests, not for real user use cases. The `event` object is always passed as it is directly passed from the native event, and looking through the [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) and [typing](https://github.com/microsoft/TypeScript/blob/main/src/lib/dom.generated.d.ts#L14376) the `key` property seems to have always some value.

Reply:
I can't find a reason why it's there.